src/Microsoft.PowerShell.Security/security/SecureStringCommands.cs

```diff
@@ -139,6 +139,12 @@ public SecureString SecureString
             }
         }
 
+        /// <summary>
+        /// Gets or sets a switch to get the secure string as plain text.
+        /// </summary>
+        [Parameter(ParameterSetName = "AsPlainText")]
+        public SwitchParameter AsPlainText { get; set; }
```

Comment:
One minor thing: `ConvertTo-SecureString -AsPlainText` requires `-Force` to be specified:
```
PS> ConvertTo-SecureString -AsPlainText abc
ConvertTo-SecureString: The system cannot protect plain text input. To suppress this warning and convert the plain text to a SecureString, reissue the command specifying the Force parameter. For more information, type: get-help ConvertTo-SecureString.
```
It's more of a storngly-enforced warning. Shall we do the same to `ConvertFrom-SecureString -AsPlainText`?

Reply:
Let's discuss this in the issue

Reply:
FWIW, I feel the `-Force` parameter is not needed, and we should consider removing it as a 'plaintext' requirement from `ConvertTo-SecureString`.